requirements.txt

```diff
@@ -1,3 +1,3 @@
 ops==1.1.0
 git+https://github.com/juju-solutions/resource-oci-image@1964d748022b762b9dce6e8bb7bdf12835102c72
-git+https://github.com/canonical/serialized-data-interface.git
+git+https://github.com/canonical/serialized-data-interface@7ccc4011027b6c65b02368b70ff0562b18fd3b59
```

Comment:
there is a pypi project for the SDI, so we should probably upload a new version and point to that.

Reply:
For sure, this is just so that I can test this PR while the SDI one is in progress. Didn't know about pypi though, that's good to know

Reply:
Uploaded the latest SDI version and updated the `requirements.txt` here to point at it.